docs/design/lexical_conventions/symbolic_tokens.md

```diff
@@ -0,0 +1,185 @@
+really important role here, and we use some rules to avoid confusion:
+
+-   There can be no whitespace between a unary operator and its operand.
+-   The whitespace around a binary operator must be consistent: either there is
+    whitespace on both sides or on neither side.
+-   If there is whitespace on neither side of a binary operator, the token
+    before the operator must be an identifier, a literal, or any kind of closing
+    bracket (for example, `)`, `]`, or `}`), and the token after the operator
+    must be an identifier, a literal, or any kind of opening bracket (for
+    example, `(`, `[`, or `{`).
+
+## Details
+
+Symbolic tokens are intended to be used for widely-recognized operators, such as
+the mathematical operators `+`, `*`, `<`, and so on. Those used as operators
+would generally be expected to also be meaningful for some user-defined types,
+and should be candidates for being made overloadable once we support operator
+overloading.
+
+### Symbolic token list
+
+The following is the initial list of symbolic tokens recognized in a Carbon
+source file:
+
+| Token | Explanation                                                                                                |
+| ----- | ---------------------------------------------------------------------------------------------------------- |
+| `*`   | Indirection, multiplication, and forming pointers                                                          |
+| `&`   | Address-of or Bitwise AND                                                                                  |
+| `=`   | Assignment                                                                                                 |
+| `->`  | Return type and `p->x` equivalent to `(*p).x` (in C++)                                                     |
+| `=>`  | Match syntax                                                                                               |
+| `[]`  | Subscript                                                                                                  |
+| `()`  | Function call and function declaration                                                                     |
+| `{}`  | Struct literals, blocks of control flow statements and the bodies of definitions (classes, functions, etc) |
+| `,`   | Separate arguments in a function call, elements of a tuple, or parameters of a function declaration        |
+| `.`   | Member access                                                                                              |
+| `:`   | Scope                                                                                                      |
+
+This list is expected to grow over time as more symbolic tokens are required by
+language proposals.
+
+Note: The above list only covers up to
+[#601](https://github.com/carbon-language/carbon-lang/pull/601) and more have
+been added since that are not reflected here.
+
+### Whitespace
```

Comment:
This whole section seems to be focused on explaining the background and rationale for this part of the design, but it should be focused primarily on what the design _is_. In other words, the focus should be on what a reader needs to know in order to read and write Carbon code. See, for example, the discussion of [real number literals](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/design/lexical_conventions/numeric_literals.md#real-number-literals): it starts by explaining the syntax and semantics of real-number literals, and then at the end of the section it goes back to explain the rationale for one particularly noteworthy design choice (namely, the decision to support hexadecimal literals).

Reply:
@geoffromer I have revamped this section! Can you take a look at it!?

Reply:
It still seems kind of focused on the rationale, especially at the beginning and the end. But after thinking about it more, I'm not sure we need this section at all: the overview already explains the whitespace rules quite clearly, so I'm not sure there's much more we need to say. It would be good to at least mention the rationale behind these rules, but I think we can do it with a sentence at the end of the overview, such as "These rules enable us to use a token like `*` as a prefix, infix, and postfix operator, without creating ambiguity."

Reply:
Will make the changes